doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -209,70 +209,99 @@ infrastructure itself from the workloads compute resources.
 
 There are two types of low latency and high throughput networks required by
 telco workloads: signalling traffic workloads and user plane traffic workloads.
 Networks used for signalling traffic are more demanding than what a standard
 overlay network can handle, but still do not need the use of user space
 networking. Due to the nature of the signalling protocols used, these type of
-networks require NAT-less communication documented in infra.net.cfg.003 and will
+networks require NAT-less communication documented in `infra.net.cfg.003` and will
 need to be served by a CNI plugin with IPVLAN or MACVLAN support. On the other
 hand, the low latency, high throughput networks used for handling the user plane
 traffic require the capability to use a user space networking technology.
 
-> Note: An infrastructure can provide the possibility to use SR-IOV with DPDK as
-an additional feature and still be conformant with CNTT.
-
-> Editors note: The possibility to SR-IOV for DPDK is under discussion.
-
-The integration of SDN solutions required by `req.inf.ntw.05` should be enabled
-via their respective CNI integration.
 
-> Note: An SDN solution can manage the pod networks via the Kubernetes API or
-can have a CNI integration.
+| Requirement | Networking Solution with Multus | Networking Solution with DANM |
+|---|---|---|
+| The overlay network encapsulation protocol needs to enable ECMP in the underlay (`infra.net.cfg.002`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin |
+| NAT (`infra.net.cfg.003`) | Supported via the additional CNI plugin | Supported |
+| Security Groups (`infra.net.cfg.004`) | Not supported | Not supported <sub>(1)<sub> |
+| SFC support (`infra.net.cfg.005`) | Not relevant | Not relevant |
+| Traffic patterns symmetry (`infra.net.cfg.006`) | Not relevant | Not relevant |
+| Network resiliency (`req.inf.ntw.01`) | Supported | Supported |
+| Centrally administrated and configured (`req.inf.ntw.03`) | Supported via Kubernetes API | Supported via Kubernetes API |
+| Dual stack IPv4 and IPv6 for Kubernetes workloads (`req.inf.ntw.04`) | Supported via the additional CNI plugin | Supported |
+| Integrating SDN controllers (`req.inf.ntw.05`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin |
+| More than one networking solution (`req.inf.ntw.06`) | Supported | Supported |
+| Choose whether or not to deploy more than one networking solution (`req.inf.ntw.07`) | Supported | Supported |
+| Kubernetes network model (`req.inf.ntw.08`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin |
+| Do not interfere with or cause interference to any interface or network it does not own (`req.inf.ntw.09`) | Supported | Supported |
+| Cluster wide coordination of IP address assignment (`req.inf.ntw.10`) | Supported via the additional CNI plugin | Supported |
 
-The container based architecture must support telecom equipment networking where
```

Comment:
I think we should keep this.

Reply:
OK - but for which requirement?

Reply:
It is related to `req.gen.cnt.05`

Reply:
I'm not sure that's the intention of that requirement and even if it is, the scope of "the Architecture" doesn't include other telecom equipment.  I think if we want to keep this statement in place, we need a corresponding requirement that asks for this.